### ISSUE TITLE
Implement new reference data set

### ISSUE THREAD

**Issue by mpresteg** [nmdp-bioinformatics/ImmunogeneticDataTools#8]
Replace current reference data set with more up to date or robust frequency data.

Current reference data sets:
- http://en.wikiversity.org/wiki/HLA/Linkage_Disequilibrium/B-C_Blocks
- http://en.wikiversity.org/wiki/HLA/Linkage_Disequilibrium/DR-DQ_Blocks

**Comment by kosoegawa** [comment]
Yes, I am aware of the current data set. I will be at CHORI and discuss with Steve Mack. When we presented this at the end of DaSH, Martin also had questions about the reference table. I would like to get more updated table from Steve and/or Martin. I would like t use the table that both of them satisfies, Steve knows about this linkage better than I do, since he is specialized in population genetics. I took the current reference table to start something. I will be responsible for formatting the lookup table.
